test/visibility/private/functions/symbolResError.chpl

```diff
@@ -0,0 +1,18 @@
+module A {
+  proc aproc() {
+    writeln("public aproc");
+    aproc(1);
+  }
+
+  private proc aproc(x: int) {
+    writeln(y);
```

Comment:
I'm not understanding why this change would conceivably have changed this behavior?

Reply:
It's mostly paranoia.  I'm fairly convinced that the machinery behind the ResolveScope type goes through all symbols, even if they aren't stored in the map I'm updating, but the control flow was odd enough that I wasn't completely convinced without a test to back it up